dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -223,6 +225,12 @@ private void addCollectionEntriesToCollection(Collection collection) {
         collectionEntries.addAll(collectionWorkflowsWithVersions);
         collectionEntries.addAll(collectionServicesWithVersions);
         collectionEntries.addAll(collectionToolsWithVersions);
+        collectionEntries.forEach(entry -> {
```

Comment:
FYI, give https://stackoverflow.com/questions/2986318/how-to-map-calculated-properties-with-jpa-and-hibernate a shot. You can probably avoid the additions to the DAO but if difficult, just break it out into a different unassigned ticket.

Reply:
Just skimmed through this post, seems like `@Formula` uses SQL, not HQL, so I'm assuming it is a problem. And if I were to create a new ticket, what shall I include?

Reply:
Possibly, for a new ticket include that link, title can be something like "workflow length or labels via hibernate annotation"